### PR TITLE
APIS-4705 - Upgraded to Play 2.6, Scala 2.12 and sbt 1.3.8

### PR DESCRIPTION
**Pull request by anjumabbas5**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,14 +1,41 @@
...
+
+lazy val compile = Seq(
+  ws,
+  "uk.gov.hmrc" %% "bootstrap-play-26" % "1.7.0"
+)
+
+lazy val test = Seq(
+  "uk.gov.hmrc" %% "hmrctest" % "3.9.0-play-26" % scope,
+  "org.scalaj" %% "scalaj-http" % "2.4.0" % scope,
+  "org.scalatest" %% "scalatest" % "3.0.8" % scope,
```
Comment (AndySpaven) [inline build.sbt:26]: Suggestion - Let this scalatest be transitive based on scalatestplus-play

File: build.sbt [hunk 2]
```diff
@@ -1,14 +1,41 @@
...
 import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
 
+import scala.util.Properties
+
 lazy val appName = "api-example-microservice"
+lazy val appDependencies: Seq[ModuleID] = compile ++ test
+
+lazy val scope: String = "test, it"
+lazy val ComponentTest = config("component") extend IntegrationTest
+def componentTestFilter(name: String): Boolean = name startsWith "component"
```
Comment (AndySpaven) [inline build.sbt:16]: This probably is not needed given there are separate folders.  This used to really only be useful when everything was in the test folder.

Comment (peteslater-ee) [reply build.sbt:16]: Hmm... the component tests are not even running right now. I need to fix this.

Comment (peteslater-ee) [reply build.sbt:16]: Nope they are all good - just my machine being :hankey:

Comment (AndySpaven) [reply build.sbt]: Well that was fun to solve.

**Pull request merged**